### PR TITLE
update pseudo-cluster.md

### PR DESCRIPTION
**Pull request by pojishou**

### CODE REVIEW

**Comment by zhongjiajie** [comment]
@pojishou Thanks for the contribution, please also change english doc and dev(english and chinese doc) in https://github.com/apache/dolphinscheduler-website/tree/master/docs/en-us/dev

**Comment by zhongjiajie** [comment]
> @pojishou Thanks for the contribution, please also change english doc and dev(english and chinese doc) in https://github.com/apache/dolphinscheduler-website/tree/master/docs/en-us/dev

Please change `dev` folder as well

**Comment by zhongjiajie** [comment]
ping @pojishou again

**Comment by zhongjiajie** [comment]
Hi @pojishou are you still work on this PR?

**Comment by pojishou** [comment]
@zhongjiajie  Sorry. The docs has been updated.

File: docs/en-us/2.0.0/user_doc/guide/installation/pseudo-cluster.md [hunk 1]
```diff
@@ -149,6 +149,19 @@ dbname="dolphinscheduler"
 # Registration center address, the address of zookeeper service
 registryServers="localhost:2181"
 ```
+* Modify the environment variable in `dolphinscheduler_env.sh` file which on the 'conf/env' directory (take the relevant software installed under '/opt/soft' as an example)
```
Comment (zhongjiajie) [inline docs/en-us/2.0.0/user_doc/guide/installation/pseudo-cluster.md:152]: Change is good, but I think we should not put these contents in this file. In this folder we should only write something user have to change during their installation time.
Maybe we should add other sections to keep all our configurations in the below place or somewhere. WDYT

image

**Comment by zhongjiajie** [comment]
I thinks keep all the config parameter in installation section is not a good idea

**Pull request closed without merging**